packages/coinstac-server-core/test/executable.js

```diff
@@ -10,23 +10,23 @@ const executable = path.join(__dirname, '..', 'bin', 'coinstac-server-core');
 
 /** Fail when a child process writes to stderr */
 function failStderr(stderr, t) {
-  stderr.pipe(concatStream(data => {
+  stderr.pipe(concatStream((data) => {
     if (data.toString()) {
       t.end('writes to stderr');
     }
   }));
 }
 
-tape('executable works', t => {
+tape('executable works', (t) => {
   t.plan(2);
 
   try {
     const child = spawn(executable);
-    const errorExit = exitCode => { t.notOk(exitCode, 'exited with non-error'); };
+    const errorExit = (exitCode) => { t.notOk(exitCode, 'exited with non-error'); };
```

Comment:
Surprised the linter doesn’t bark at this brace-expression organization. Want to break it up with some `\n`s to make it easier to read?